growth/too/tasks/ref.py

```diff
@@ -0,0 +1,71 @@
+import sys
+import subprocess
+import shlex
+from subprocess import PIPE
+import numpy as np
+import tempfile
+import requests
+
+from celery.task import PeriodicTask
+from celery.utils.log import get_task_logger
+
+from ..flask import app
+from . import celery
+from .. import models
+
+log = get_task_logger(__name__)
+
+__all__ = ('ztf_references',)
+
+
+@app.task(base=PeriodicTask, shared=False, run_every=3600)
+def ztf_references():
+
+    user = app.config['IRSA_USERNAME']
+    pw = app.config['IRSA_PASSWORD']
```

Comment:
I find that the simulated browser login is not needed for querying metadata with the TAP interface. For TAP, stashing the username and password in the netrc fiile is sufficient:

    machine irsa.ipac.caltech.edu login XXXXXX password XXXXXX